### ISSUE TITLE
Unicode source support

### ISSUE THREAD

**Issue by dstasio** [x13pixels/remedybg-issues#205]
It would be nice if remedy could render utf-8 characters from source files.
Also, if a variable name contains a unicode character, the watch window is unable to parse it.

Here's the source:
![image](https://user-images.githubusercontent.com/22714616/103678960-81fc3900-4f84-11eb-98c7-0deba3c8b519.png)

And remedy displays it as
![image](https://user-images.githubusercontent.com/22714616/103679112-bbcd3f80-4f84-11eb-9205-b80ac66303df.png)
![image](https://user-images.githubusercontent.com/22714616/103679137-c556a780-4f84-11eb-863f-23918debdbaa.png)

**Comment by skeeto** [comment]
Related: a UTF-8 BOM displays as a `?` at the very beginning of the source. (I prefer a BOM to the `cl` `/utf-8` switch.)

**Comment by SanJacobs** [comment]
Yeah, this is currently getting in my way. Would love to see it fixed.
